### PR TITLE
Update settings watches api and sentinel logs in e2e tests

### PR DESCRIPTION
**Pull request by dianabarsan**
# Description

Stabilize and reduce duration of e2e tests that require updating settings and interacting with an API endpoint or when expecting sentinel to process docs in a certain way.
Speeds up the e2e test job by ~ 1.5 minutes (on GHA)

https://github.com/medic/angular10-migration/issues/160

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by dianabarsan** [comment]
hey @latin-panda . Would you mind having a look, please? 
Sorry for the high line count, it's mostly repetitive in adding an additional parameter to a call. 
I've been wanting to do this for at least 1.5 years!

**Comment by dianabarsan** [comment]
hey @njogz 
Just noticed Jennifer is on vacation. Would you, please, have a look at this? Thanks for your time!

File: tests/utils.js [hunk 1]
```diff
@@ -499,30 +518,47 @@ module.exports = {
   /**
    * Update settings and refresh if required
    *
-   * @param      {Object}   updates  Object containing all updates you wish to
-   *                                 make
-   * @param      {Boolean}  ignoreRefresh  don't bother refreshing
-   * @return     {Promise}  completion promise
+   * @param {Object}         updates  Object containing all updates you wish to
+   *                                  make
+   * @param  {Boolean|String} ignoreReload if false, will wait for reload modal and reload. if truthy, will tail
```
Comment (njogz) [inline tests/utils.js:523]: I was confused when I saw this being called with a boolean then a string so having this annotation helps.

**Review by njogz — APPROVED** [review]
Looks good.

File: tests/utils.js (hunk 1 above)
Comment (dianabarsan) [reply tests/utils.js:523]: Yea, I've been toying around with the idea of making this API different. 
One idea was to always need to pass a string to target a service, but that would require lots of edits where this method is called. 
I'm not sure what is best, so I went with the approach that required least edits.

**Pull request merged**